src/components/slider/Slider.tsx

```diff
@@ -100,7 +100,7 @@ export const Slider = ({ min, max, value, step, onChange }: SliderProps) => (
     <StyledSlider
       max={max}
       min={min}
-      onValueChange={([val]) => onChange(val)}
+      onValueChange={([val]) => val !== undefined && onChange(val)}
```

Comment:
```suggestion
      onValueChange={([val]) => !!val && onChange(val)}
```
I think you can do this instead 🤔

Reply:
It is a number, which could be 0 so I need to explicitly check for only undefined.

Reply:
I just looked deeper into it, the type of `[val]` claimed that it's always a number, would it ever be `undefined`?

Reply:
It is an indexed access (array destructure) so it can always be undefined

Reply:
Got it. All of these changes are because you turned on the ts rule for index accessing and it's stricter checking 😆 That's nice!!!